projects/ion/src/lib/smart-table/smart-table.component.html

```diff
@@ -137,53 +137,31 @@
                   <ion-button
                     *ngIf="
                       action.confirm &&
-                      !config.tooltipConfig &&
-                      (action.show ? !showAction(row, action) : true)
-                    "
-                    ionPopConfirm
-                    [ionPopConfirmTitle]="action.confirm.title"
```

Comment:
Will this removal not affect the current functioning of confirm?

Reply:
It won't, there were three buttons only because we could not show the tooltip conditionaly, it didnt affected the popconfirm behavior. With #747 implementation we dont have to do this anymore and have the previous code with only two buttons depending of action.confirm.

Reply:
confirm is working